Resources/Prototypes/Entities/Objects/Weapons/Guns/Shotguns/shotguns.yml

```
@@ -72,7 +72,9 @@
   - type: AmmoCounter
   - type: GunRequiresWield #remove when inaccuracy on spreads is fixed
   - type: Gun
-    fireRate: 2
+    #Moffstation - Begin Bulldog buff
+    fireRate: 2.5
+    #Moffstation - end
```

```suggestion
    fireRate: 2.5 # Moffstation - Bulldog buff
```
Single lines can have tail comments instead of wrapping blocks.